src/selectors/authentication.js

```diff
@@ -1,5 +1,27 @@
 import { createSelector } from "reselect";
+import { platformRoles } from "./../constants";
```

Comment:
Is there a reason for the ./?  Doesn't .. target current directory?

Reply:
Just my habbit. Some time ago I found that there is no one "correct" rule in conventions to specify relative paths. Some people like to write relative path implicitly starting from "current directory" others just goes to the top without that. 

Since we have lot of that accross our projects I would prefer to leave things as they are. And if we would decide our internal "rules" about that, then we need to replace it in many places.